servers/publikator/lib/github/index.js

```diff
@@ -366,17 +365,26 @@ module.exports = {
       .then(tags =>
         uniqWith(tags, (a, b) => a.name === b.name)
       )
+      .catch(e => {})
```

Comment:
shouldn't we log this?

Reply:
I added logging there. It's going to fire every time github is down, but I guess that's ok.